obspy/io/gcf/libgcf.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+# reads Guralp Compressed Format (GCF) Files
+# By Ran Novitsky Nof @ BSL, 2016
+# ran.nof@gmail.com
+# Based on Guralp's GCF reference (GCF-RFC-GCFR, Issue C, 2011-01-05)
+# more details available from: http://www.guralp.com/apps/ok?doc=GCF_Intro
+# last access: June, 2016
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import numpy as np
+
+from obspy import UTCDateTime
+
+sps_d = {  # Table 3.1: special sample rates
+    157: 0.1,
+    161: 0.125,
+    162: 0.2,
+    164: 0.25,
+    167: 0.5,
+    171: 400,
+    174: 500,
+    176: 1000,
+    179: 2000,
+    181: 4000}
+time_offsets_d = {  # Table 3.1: Time fractional offset denominator
+    171: 8.,
+    174: 2.,
+    176: 4.,
+    179: 8.,
+    181: 16.}
+compression_d = {  # Table 3.2: format field to data type
+    1: '>i4',
+    2: '>i2',
+    4: '>i1'}
+bits_d = {  # Table 3.2: format field to data bytes
+    1: 4,  # 32bits
+    2: 2,  # 16bits
+    4: 1}   # 8bits
+
+
+def is_gcf(f):
+    """
+    Test if file is GCF by reading at least 1 data block
+    """
+    header, data = read_data_block(f)
+
+
+def decode36(data):
+    """
+    Decode base 36 data
```

Comment:
I know that's what that site says, but the terminology is sort of weird. It's _encoding_ the (base 10 or base 2, depending how you look at it) number into base 36 to get a string.

One could say that you are "decoding the bytes into a string" in Python parlance (except that this function only does half of that process), but then it wouldn't be _from_ base 36; I'm not sure what the correct term for that is.